app/ui-react/syndesis/src/app/App.tsx

```diff
@@ -201,6 +211,31 @@ export class App extends React.Component<IAppBaseProps, IAppBaseState> {
                           <WithServerEvents apiUri={apiUri} headers={headers}>
                             {functions => (
                               <ServerEventsContext.Provider value={functions}>
+                                <WithApiVersion>
```

Comment:
Does this poll?  If it does, then it'll be polling all the time, and if it doesn't it'll run when the page loads up.

Reply:
You could also probably do `{this.state.showAboutModal && (<WithApiVersion>` here actually, that'd mean the call would be made when the modal is rendered on the page.